src/Files.App/BaseLayout.cs

```diff
@@ -897,18 +897,22 @@ protected void FileList_ContainerContentChanging(ListViewBase sender, ContainerC
 
 		private void RefreshContainer(SelectorItem container, bool inRecycleQueue)
 		{
-			container.PointerEntered -= ListedItem_PointerEntered;
-			container.PointerExited -= ListedItem_PointerExited;
 			container.PointerPressed -= FileListItem_PointerPressed;
+			container.PointerEntered -= FileListItem_PointerEntered;
+			container.PointerExited -= FileListItem_PointerExited;
+
 			if (inRecycleQueue)
 			{
 				UninitializeDrag(container);
 			}
 			else
 			{
-				container.PointerEntered += ListedItem_PointerEntered;
-				container.PointerExited += ListedItem_PointerExited;
 				container.PointerPressed += FileListItem_PointerPressed;
+				if (UserSettingsService.PreferencesSettingsService.SelectFilesOnHover)
```

Comment:
It appears that we need to listen when this property changes as well, otherwise the user needs to restart the app for the setting to take effect.

Reply:
True. Refreshing the file list should be enough though.

Reply:
Can probably do the same thing we do when users change the setting to show hidden items.